content/en/account_management/billing/containers.md

```diff
@@ -23,6 +23,33 @@ Fargate is charged based on the concurrent number of tasks. For pricing, see the
 
 Billing of [GKE Autopilot][5] environments is the same as that of [GKE Standard][6].
 
+## Frequently asked questions
+
+**How does Datadog measure on-demand usage?**
+
+Containers are metered in five minute increments. The difference is calculated between the number of containers observed and the allotment for the account. The allotment for the account is the total of the included containers (10/host for Pro and 20/host for Enterprise) and any container commitment. The containers exceeding the allotment are averaged out over the hour to calculate the hourly on-demand usage. This method normalizes the on-demand usage for short term spikes and variations between hosts.
+
+**What if I run many containers for a short period of time?**
```

Comment:
Let's avoid using "I"

Also, "many" sounds awkward here.

"What if a user runs an abnormal number of containers for a short period of time?"